src/test/library/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(test_library)
+
+add_executable(test_library "")
+target_sources(test_library
+    PUBLIC pch.cpp text_writer.cpp)
+
+target_include_directories(test_library
+    PUBLIC ${XLANG_LIBRARY_PATH} ${XLANG_TEST_INC_PATH})
+
+CONSUME_PAL(test_library)
```

Comment:
Does this library test really depend on the PAL?

Reply:
No, good catch. I just copied the test_platform project and missed this. My cmake skills are nonexistent.